Example/Podfile.lock

```diff
@@ -1,7 +1,7 @@
 PODS:
-  - PrimerSDK (1.28.0-beta.4):
-    - PrimerSDK/Core (= 1.28.0-beta.4)
-  - PrimerSDK/Core (1.28.0-beta.4)
+  - PrimerSDK (1.29.2):
```

Comment:
I released 1.29.2 this morning 😆

Please update to 1.29.3

Reply:
at the time of writing this comment, I cannot see the new version on git 😅 
<img width="394" alt="Screen Shot 2022-03-16 at 17 00 20" src="https://user-images.githubusercontent.com/3584008/158633533-1ee267c3-40c0-4b27-9aff-baadec397904.png">
<img width="340" alt="Screen Shot 2022-03-16 at 17 00 11" src="https://user-images.githubusercontent.com/3584008/158633546-f51f463a-17e2-4c1e-a471-bf6e3690a528.png">